modules/nw-endpoint-route53.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-government-region.adoc
+
+[id="nw-endpoint-route53_{context}"]
+= Endpoint configuration for route53
+
+If you install in either GovCloud region, the Ingress Operator uses `us-gov-west-1` region for Route53 and tagging API clients.
```

Comment:
Checked https://bugzilla.redhat.com/show_bug.cgi?id=1866299 (Comment 40 and 58) , for Route53 it always uses `https://route53.us-gov.amazonaws.com`, but tagging API follows the region name and can use `https://tagging.us-gov-east-1.amazonaws.com` or `https://tagging.us-gov-west-1.amazonaws.com`.

Reply:
To avoid confusion, we should tell user the difference between the endpoints user specified in yaml file and the ones Ingress operator uses in implementation. 
And user should specify Route53 as `https://route53.us-gov.amazonaws.com` for both GovCloud region, see https://docs.aws.amazon.com/govcloud-us/latest/UserGuide/using-govcloud-endpoints.html, only US-WEST has the endpoints.

Reply:
Thank you, @lihongan! I added call outs to the yaml file example, would you mind taking a look at those please?